src/navbar/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { AppBar, Grid } from '@material-ui/core'
+
+import { Title } from 'navbar/title'
+import { UserContent } from 'navbar/user-content'
+import { useStyles } from 'navbar/style'
+
+export const Navbar = () => <AppBar position={'static'}>
+  <Grid container className={useStyles().root} alignContent={'center'}>
```

Comment:
Try to  extract `<Grid container className={useStyles().root} alignContent={'center'}>` in a separate AppBarWrapper 

```js
export const AppBarWrapper = ({children}) => <Grid container className={useStyles().root} alignContent={'center'}>
{children}
</Grid>
```

This will make it have more reactive feal. 
```js
<AppBar>
  <AppBarWrapper>
    <Title />
    <UserContent />
   </AppBarWrapper>
</AppBar>
```

Reply:
Done